### DISCUSSION TITLE
Alternative keywords

### DISCUSSION THREAD

**Discussion by actopozipc** [actopozipc/German-Python-Interpreter#6] [Polls]
Because some translations are ambiguous, would you like to have multiple possible translations for a keyword? E.g. "während" as well as "solange" valid for While?

**Comment by AntonPieper** [comment]
As a programming language I think ambiguity would be counterproductive as it could lead to unexpected errors, however in the German language there absolutely are multiple ways of saying the same thing. If multiple translations don't interfere with other Syntax, I am fine with it. Just need to make sure one doesn't shoot themselves in the foot by introducing keywords in one place early on that could have better been used elsewhere

**Comment by Mythe8** [comment]
Look at my changes within the Schlange.py file
Also change _brechen_ to _breche_ on the README file
